crates/iota-genesis-builder/examples/snapshot_only_test_outputs.rs

```diff
@@ -32,6 +29,8 @@ enum Snapshot {
     Iota {
         #[clap(long, help = "Path to the Iota Hornet full-snapshot file")]
         snapshot_path: String,
+        #[clap(long, help = "Specify the delegator address")]
+        delegator: String,
```

Comment:
Assume we can set it as IotaAddress, but it’s really minor(because it’s an example).

Reply:
Yes, you're right. Done